config/filament-invite.php

```diff
@@ -5,6 +5,6 @@
     'expire' => 60 * 24 * 2, // 2 days
 
     'routes' => [
-        'reset' => 'password.reset',
+        'reset' => null,
     ],
```

Comment:
We can add a new config option to handle the base URL selection:

```php
// URL Configuration
'use_panel_url' => false, // Use panel URL instead of app.url for custom routes (default: false for backward compatibility)
```